src/components/swap/AdvancedSwapDetails.test.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  TEST_ALLOWED_SLIPPAGE,
+  TEST_TOKEN_1,
+  TEST_TRADE_EXACT_INPUT,
+  TEST_TRADE_EXACT_OUTPUT,
+  toCurrencyAmount,
+} from 'test-utils/constants'
+import { fireEvent, render, screen } from 'test-utils/render'
```

Comment:
apparently fireEvent is actually not the best tool for the job here: https://testing-library.com/docs/user-event/intro/

Reply:
this should be fixed soon! once it's fixed I'll ping you and ask you to merge main into these PRs so that they can be tested locally - as-is when I try to test locally it takes many minuntes :(

Reply:
thanks its so much faster now! changed to user event.